### DISCUSSION TITLE
keyball61で v1.2.0にアップデートしたらトラックボールが効かなくなった

### DISCUSSION THREAD

**Discussion by u1tnk** [Yowkees/keyball#377] [Q&A]
本日 v1.2.0にアップデートしたところ、トラックボール、マウスボタンが反応しなくなりました。

アップデート時旧ファーム用のレイアウトをrestoreしたところ、マウスボタンも効きませんでしたが、こちらは再設定で動作しました。
ですが、トラックボールを動かしてもカーソルは動きませんでした。(LED表示確認するべきなのは今気づいたので確認できておりません 🙏 )

何か v1.2.0アップデート時に remap-keys.app からFLASHする以外に必要な事がありますでしょうか？

元のバージョンはっきりしませんが、 v1.1.1のファームに戻したところ動作するようになりましたので困ってはおりませんが、報告までと思いました。

動作確認など必要でしたら言ってください。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

トラックボールが動かなくなる事象は初めて聞きました。

v1.2.0へのアップデートはREMAPのキーボードカタログに登録してあるファームウェアを書き込むだけでOKです。

OLEDのカーソル移動量も動きませんか？
また、Keyball種類や書き込んだソフト、書き込み方法を教えください。

以上、よろしくお願いいたします。

**Reply by u1tnk** [reply]
回答ありがとうございます。

> OLEDのカーソル移動量も動きませんか？

はい、さきほどもう一回1.2.0に上げてみて確認しましたが、OLEDのballのところの数値はボールを回してみても変わりませんでした。
書き込み直後、ケーブルさし直し後に試しても同様でした。

> Keyball種類

keyball 61。 2023年3月26日に購入しました。


> 書き込んだソフト、書き込み方法を教えください。
 
REMAPのキーボードカタログから 1.2.0 を デフォルトのcaterina で書きこみました。

**Comment by Yowkees** [comment]
ご確認ありがとうございます。
REMAPのキーボードカタログから書き込んでもOLEDの数値に変化がなしですか。センサーが反応していないということですね。
当方でもさきほどREMAPからKeyball61へファームウェアを書き込んでみましたが問題なく動作しております。
こちらで再現できないので、ProMicroの種類やPC環境に依存しているでしょうか。
現状では追及が難しそうですので、同様の報告があったときに対処させていただきます。申し訳ありません。

**Reply by u1tnk** [reply]
なるほど。それは仕方ないですね。

当面1.1.1で問題は無いのでしばらく様子見ます。

ご確認ありがとうございました。
